tests/shared/src/test/scala/scala/meta/tests/tokenizers/TokenizerSuite.scala

```diff
@@ -2102,5 +2102,40 @@ class TokenizerSuite extends BaseTokenizerSuite {
                     |""".stripMargin.nl2lf
     assertTokenizedAsStructureLines(code, struct)
   }
+  test("broken-interpolator") {
+    val code = """|
+                  |s'''|Available languages (default = $):
+                  |verilog - Verilog or SystemVerilog dialects
+                  |vhdl    - VHDL dialects
+                  |
+                  |Available Verilog/SystemVerilog dialects (default = $defaultDialect):
+                  |v2001   - Verilog 2001
+                  |sv2005  - 
+                  |'''.stripMargin
+                  |""".stripMargin.replace("'''", "\"\"\"")
+
+    val struct = """|BOF [0..0)
+                    |LF [0..1)
+                    |Interpolation.Id(s) [1..2)
+                    |Interpolation.Start(''') [2..5)
+                    |Interpolation.Part(|Available languages (default = ) [5..37)
+                    |Interpolation.SpliceStart [37..38)
+                    |Invalid(Not one of: `$'_, `$$', `$'ident, `$'this, `$'BlockExpr) [38..38)
+                    |Invalid():
```

Comment:
```suggestion
                    |StringPart():
```
alternatively we could try having string part and just have the single invalid token.